cmd/minishift/cmd/start.go

```diff
@@ -243,6 +244,31 @@ func applyAddOns(ip string, routingSuffix string, ocPath string, kubeConfigPath
 	}
 }
 
+// TODO - persistent volume creation should really be fixed upstream, aka 'cluster up'. See https://github.com/openshift/origin/issues/14076 (HF)
+// configurePeristentVolumes makes sure that the default persistent volumes created by 'cluster up' have the right permissions - see https://github.com/minishift/minishift/issues/856
+func configurePeristentVolumes(sshCommander provision.SSHCommander) {
+	fmt.Print("-- Waiting for persistent volumes to be created ... ")
+
+	hostPvDir := viper.GetString(hostPvDir)
+
+	// there are in total 101 directories when using ls. There is an additonal 'registry' directory
```

Comment:
There is no other good option. There are several issues open for cluster up which will improve on this. In particular the ability to control the number of pas created as well as not creating them in the background. Once this is available we will adjust this code.